### PR TITLE
Update ReadMe to standard OpenSRP format

### PR DESCRIPTION
**Pull request by ndegwamartin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,59 @@
...
+-    Family profile for managing families and fmily members
+-  Task register for displaying Available tasks for a given location in a given plan.
+-  Map for displaying operational areas and structures within it
+-  Support for FI (focus investigation), MDA (Mass drug adminstartion) and IRS (Indoor Residue Spray) plans
+-  Task based color coding for both map view and task register
+-  Downloading of offline maps
+-  P2P sync
+-  Base map layer switcher (switching between Mapbox and Digital globe maps)
+ -  Dynamic form configuration
+-  Supports Editing focus area boundaries
```
Comment (keymane) [inline README.md:38]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/manual/OpenSRP/opensrp-client-reveal/pullRequest?prid=5798757)

**Comment by keymane** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 2
- Added 13
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/OpenSRP/opensrp-client-reveal/pullRequest?prid=5798757&bid=18980342)

File: README.md [hunk 2]
```diff
@@ -1,5 +1,59 @@
...
+## Deployment
+[Production releases](https://smartregister.atlassian.net/wiki/spaces/Documentation/pages/1141866503/How+to+create+a+release+APK)
+
+## Features
+-  Supports tasking functionality
+-    Family profile for managing families and fmily members
+-  Task register for displaying Available tasks for a given location in a given plan.
+-  Map for displaying operational areas and structures within it
+-  Support for FI (focus investigation), MDA (Mass drug adminstartion) and IRS (Indoor Residue Spray) plans
+-  Task based color coding for both map view and task register
```
Comment (keymane) [inline README.md:33]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/manual/OpenSRP/opensrp-client-reveal/pullRequest?prid=5798757)

File: README.md [hunk 3]
```diff
@@ -1,5 +1,59 @@
...
+
+## Deployment
+[Production releases](https://smartregister.atlassian.net/wiki/spaces/Documentation/pages/1141866503/How+to+create+a+release+APK)
+
+## Features
+-  Supports tasking functionality
+-    Family profile for managing families and fmily members
+-  Task register for displaying Available tasks for a given location in a given plan.
+-  Map for displaying operational areas and structures within it
+-  Support for FI (focus investigation), MDA (Mass drug adminstartion) and IRS (Indoor Residue Spray) plans
```
Comment (keymane) [inline README.md:32]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/manual/OpenSRP/opensrp-client-reveal/pullRequest?prid=5798757)

File: README.md [hunk 4]
```diff
@@ -1,5 +1,59 @@
...
+
+[Android client unit tests](https://smartregister.atlassian.net/wiki/spaces/Documentation/pages/65570428/OpenSRP+Client)
+
+## Deployment
+[Production releases](https://smartregister.atlassian.net/wiki/spaces/Documentation/pages/1141866503/How+to+create+a+release+APK)
+
+## Features
+-  Supports tasking functionality
+-    Family profile for managing families and fmily members
+-  Task register for displaying Available tasks for a given location in a given plan.
```
Comment (keymane) [inline README.md:30]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/manual/OpenSRP/opensrp-client-reveal/pullRequest?prid=5798757)

File: README.md [hunk 5]
```diff
@@ -1,5 +1,59 @@
...
+-  Map for displaying operational areas and structures within it
+-  Support for FI (focus investigation), MDA (Mass drug adminstartion) and IRS (Indoor Residue Spray) plans
+-  Task based color coding for both map view and task register
+-  Downloading of offline maps
+-  P2P sync
```
Comment (keymane) [inline README.md:35]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/manual/OpenSRP/opensrp-client-reveal/pullRequest?prid=5798757)

File: README.md [hunk 6]
```diff
@@ -1,5 +1,59 @@
...
+ -  Dynamic form configuration
+-  Supports Editing focus area boundaries
+
+## Versioning
+We use SemVer for versioning. For the versions available, see the tags on this repository.
+For more details check out <https://semver.org/>
+
+## Authors/Team 
+-  The OpenSRP team
+-  See the list of contributors who participated in this project from the [Contributors](../../graphs/contributors) link
```
Comment (keymane) [inline README.md:46]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/manual/OpenSRP/opensrp-client-reveal/pullRequest?prid=5798757)

File: README.md [hunk 7]
```diff
@@ -1,5 +1,59 @@
...
+[Android client unit tests](https://smartregister.atlassian.net/wiki/spaces/Documentation/pages/65570428/OpenSRP+Client)
+
+## Deployment
+[Production releases](https://smartregister.atlassian.net/wiki/spaces/Documentation/pages/1141866503/How+to+create+a+release+APK)
+
+## Features
+-  Supports tasking functionality
+-    Family profile for managing families and fmily members
+-  Task register for displaying Available tasks for a given location in a given plan.
+-  Map for displaying operational areas and structures within it
```
Comment (keymane) [inline README.md:31]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/manual/OpenSRP/opensrp-client-reveal/pullRequest?prid=5798757)

File: README.md [hunk 8]
```diff
@@ -1,5 +1,59 @@
...
+-  Support for FI (focus investigation), MDA (Mass drug adminstartion) and IRS (Indoor Residue Spray) plans
+-  Task based color coding for both map view and task register
+-  Downloading of offline maps
```
Comment (keymane) [inline README.md:34]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/manual/OpenSRP/opensrp-client-reveal/pullRequest?prid=5798757)

File: README.md [hunk 9]
```diff
@@ -1,5 +1,59 @@
...
+-  P2P sync
+-  Base map layer switcher (switching between Mapbox and Digital globe maps)
```
Comment (keymane) [inline README.md:36]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/manual/OpenSRP/opensrp-client-reveal/pullRequest?prid=5798757)

File: README.md [hunk 10]
```diff
@@ -1,5 +1,59 @@
...
+ -  Dynamic form configuration
```
Comment (keymane) [inline README.md:37]: Codacy found an issue: [[list-item-bullet-indent] Incorrect indentation before bullet: remove 1 space](https://app.codacy.com/manual/OpenSRP/opensrp-client-reveal/pullRequest?prid=5798757)

File: README.md [hunk 11]
```diff
@@ -1,5 +1,59 @@
...
+### Running the tests
+
+[Android client unit tests](https://smartregister.atlassian.net/wiki/spaces/Documentation/pages/65570428/OpenSRP+Client)
+
+## Deployment
+[Production releases](https://smartregister.atlassian.net/wiki/spaces/Documentation/pages/1141866503/How+to+create+a+release+APK)
+
+## Features
+-  Supports tasking functionality
+-    Family profile for managing families and fmily members
```
Comment (keymane) [inline README.md:29]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/manual/OpenSRP/opensrp-client-reveal/pullRequest?prid=5798757)

File: README.md [hunk 12]
```diff
@@ -1,5 +1,59 @@
...
+-  Base map layer switcher (switching between Mapbox and Digital globe maps)
+ -  Dynamic form configuration
+-  Supports Editing focus area boundaries
+
+## Versioning
+We use SemVer for versioning. For the versions available, see the tags on this repository.
+For more details check out <https://semver.org/>
+
+## Authors/Team 
+-  The OpenSRP team
```
Comment (keymane) [inline README.md:45]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/manual/OpenSRP/opensrp-client-reveal/pullRequest?prid=5798757)

File: README.md [hunk 13]
```diff
@@ -1,5 +1,59 @@
...
+
+### Running the tests
+
+[Android client unit tests](https://smartregister.atlassian.net/wiki/spaces/Documentation/pages/65570428/OpenSRP+Client)
+
+## Deployment
+[Production releases](https://smartregister.atlassian.net/wiki/spaces/Documentation/pages/1141866503/How+to+create+a+release+APK)
+
+## Features
+-  Supports tasking functionality
```
Comment (keymane) [inline README.md:28]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/manual/OpenSRP/opensrp-client-reveal/pullRequest?prid=5798757)

**Comment by keymane** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 2
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/OpenSRP/opensrp-client-reveal/pullRequest?prid=5798757&bid=18980342)

File: README.md [hunk 14]
```diff
@@ -1,5 +1,59 @@
...
+-   Dynamic form configuration
+-   Supports Editing focus area boundaries
+
+## Versioning
+We use SemVer for versioning. For the versions available, see the tags on this repository.
+For more details check out <https://semver.org/>
+
+## Authors/Team 
+-   The OpenSRP team
+-  See the list of contributors who participated in this project from the [Contributors](../../graphs/contributors) link
```
Comment (keymane) [inline README.md:46]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/manual/OpenSRP/opensrp-client-reveal/pullRequest?prid=5798757)

**Comment by keymane** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/OpenSRP/opensrp-client-reveal/pullRequest?prid=5798757&bid=18980342)

File: README.md [hunk 15]
```diff
@@ -1,5 +1,59 @@
...
+### Running the tests
+
+[Android client unit tests](https://smartregister.atlassian.net/wiki/spaces/Documentation/pages/65570428/OpenSRP+Client)
+
+## Deployment
+[Production releases](https://smartregister.atlassian.net/wiki/spaces/Documentation/pages/1141866503/How+to+create+a+release+APK)
+
+## Features
+-   Supports tasking functionality
+-   Family profile for managing families and fmily members
```
Comment (Rkareko) [inline README.md:29]: There is a typo in the word `fmly`

**Comment by keymane** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/OpenSRP/opensrp-client-reveal/pullRequest?prid=5798757&bid=18980342)

**Pull request merged**